charts/traction/templates/ui/configmap.yaml

```diff
@@ -13,15 +13,15 @@ data:
   FRONTEND_INNKEEPER_OIDC_LABEL: {{ .Values.ui.oidc.label | quote }}
   FRONTEND_INNKEEPER_SHOW_ADMIN: {{ .Values.ui.oidc.showInnkeeperAdminLogin | quote }}
   FRONTEND_TENANT_SHOW_WRITABLE_COMPONENTS: {{ .Values.ui.oidc.showWritableComponents | quote }}
-  FRONTEND_TENANT_PROXY_URL: http://{{ include "tenant_proxy.fullname" . }}
+  FRONTEND_TENANT_PROXY_URL: https://{{ include "tenant_proxy.host" . }}
   SERVER_OIDC_JWKS: {{ .Values.ui.oidc.jwksUri | quote }}
   SERVER_OIDC_REALM: {{ .Values.ui.oidc.realm | quote }}
   SERVER_OIDC_ROLE: {{ .Values.ui.oidc.roleName | quote }}
   SERVER_SMTP_INNKEEPER_INBOX: {{ .Values.ui.smtp.innkeeperInbox | quote }}
   SERVER_SMTP_PORT: {{ .Values.ui.smtp.port | quote }}
   SERVER_SMTP_SENDER_ADDRESS: {{ .Values.ui.smtp.senderAddress | quote }}
   SERVER_SMTP_SERVER: {{ .Values.ui.smtp.server | quote }}
-  SERVER_TRACTION_URL: http://{{ include "tenant_proxy.fullname" . }}
+  SERVER_TRACTION_URL: https://{{ include "tenant_proxy.host" . }}
```

Comment:
@loneil is there a reason to have two environment variables set with the same value? Would we ever have the values be different (i.e.: can we have `SERVER_TRACTION_URL` route internally using the service name?), or should we collapse them into one environment variable?

Reply:
Thinking about it I'd keep it as it is rather than changing anything. One is amongst the group of frontend configuration for the client side app, the other is for the Node API.
So while they're the same value (and likely to be the same value) I think having one shared configuration while the rest are split between the 2 apps would be more confusing.

As well, there could be cases to have them behave differently, as Ivan suggested earlier could have the API one be an internal route sometime. Also if we want to split the client side SPA to a different serving paradigm (like Caddy or something rather than being served by the same Node API) then they wouldn't have the same env vars.